apps/ui/src/components/ButtonFollow.vue

```diff
@@ -12,7 +16,9 @@ const followedSpacesStore = useFollowedSpacesStore();
 const { web3 } = useWeb3();
 
 const spaceFollowed = computed(() => followedSpacesStore.isFollowed(spaceIdComposite));
-const hidden = computed(() => web3.value?.type === 'argentx');
+const hidden = computed(
+  () => web3.value?.type === 'argentx' || SUPPORTED_NETWORKS.includes(props.space.network)
```

Comment:
This should be reversed as it hides the button.

Reply:
😒